managed/models/agent_model.go

```diff
@@ -339,10 +339,10 @@ func (s *Agent) DSN(service *Service, dsnParams DSNParams, tdp *DelimiterPair) s
 		cfg.Params = make(map[string]string)
 		if s.TLS {
 			switch {
-			case s.TLSSkipVerify:
-				cfg.Params["tls"] = skipVerify
 			case len(s.Files()) != 0:
 				cfg.Params["tls"] = "custom"
+			case s.TLSSkipVerify:
+				cfg.Params["tls"] = skipVerify
```

Comment:
It was also mandatory to have "custom" as the first case.

Reply:
could you please add it as a comment there?

Reply:
and also add that TLSSkipVerify for custom is handled in pmm-agent side

Reply:
@BupycHuk Sure. Added on both places (exporter, QAN agents) :
https://github.com/percona/pmm/pull/2909/files#diff-55a051531f2bd4996142e7a3723954354b3c3e7ae921ca0e5b024178d9550f3aR341-R342

https://github.com/percona/pmm/pull/2909/files#diff-55a051531f2bd4996142e7a3723954354b3c3e7ae921ca0e5b024178d9550f3aR372-R373